packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-configure-oidc.cy.ts

```diff
@@ -0,0 +1,158 @@
+describe("TESTS: Configuring Elytron OIDC client in webconsole", () => {
+  const oidcAddress = ["subsystem", "elytron-oidc-client"];
+  const providerAddress = ["subsystem", "elytron-oidc-client", "provider", "keycloak"];
+  const secureDeploymentAddress = ["subsystem", "elytron-oidc-client", "secure-deployment", "wildfly-console"];
+  const secureServerAddress = ["subsystem", "elytron-oidc-client", "secure-server", "wildfly-management"];
+  const coreSeviceRbacAddress = ["core-service", "management", "access", "authorization"];
+  const mappedRoleAddress = [
+    "core-service",
+    "management",
+    "access",
+    "authorization",
+    "role-mapping",
+    "Administrator",
+    "include",
+    "user-userwithmappedrole",
+  ];
+  const optionalFields = "a.field-section-toggle-pf";
+  const activationRBACWarning = "#hal-finder-preview > div:nth-child(2) > a";
+  const activationRBACWizadrWarning = "#hal-modal > div > div > div.modal-body > p > p:nth-child(2)";
```

Comment:
> `#hal-modal > div > div > div.modal-body > p > p:nth-child(2)`

Do we really need to be so much precise? Isn't something like `#hal-modal` or `#hal-modal div.modal-body` enough?

Reply:
No, i also try `#hal-modal > div.modal-body > p > p:nth-child(2)` and it doesn't work.